### PR TITLE
Configuration PetProject

### PR DESCRIPTION
**Pull request by RafaelKhammadeev**
- User should be able to sign out
- User should be able to sign in

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -50,12 +61,14 @@ gem "bootsnap", require: false
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
-  gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem 'debug', platforms: %i[mri mingw x64_mingw]
```
Comment (RasimKhusaenov) [inline Gemfile:64]: I think you should choose between pry and debug

File: Procfile.dev [hunk 2]
```diff
@@ -0,0 +1,2 @@
+web: unset PORT && bin/rails server
+css: yarn build:css --watch
```
Comment (RasimKhusaenov) [inline Procfile.dev:2]: Why you need this?

File: Gemfile [hunk 3]
```diff
@@ -1,34 +1,36 @@
-source "https://rubygems.org"
+# frozen_string_literal: true
```
Comment (RasimKhusaenov) [inline Gemfile:1]: Do you really need this?

File: app/controllers/posts_controller.rb [hunk 4]
```diff
@@ -1,70 +1,58 @@
...
-    respond_to do |format|
-      if @post.save
-        format.html { redirect_to post_url(@post), notice: "Post was successfully created." }
-        format.json { render :show, status: :created, location: @post }
-      else
-        format.html { render :new, status: :unprocessable_entity }
-        format.json { render json: @post.errors, status: :unprocessable_entity }
-      end
+    if @post.save
+      redirect_to post_url(@post), notice: 'Post was successfully created.'
```
Comment (RasimKhusaenov) [inline app/controllers/posts_controller.rb:26]: How about just
```suggestion
      redirect_to @post, notice: 'Post was successfully created.'
```
?

File: app/views/application/_navigation.html.erb [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+</body>
+</html>
```
Comment (RasimKhusaenov) [inline app/views/application/_navigation.html.erb:10]: Do you need this file?

File: bin/dev [hunk 6]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+if ! gem list foreman -i --silent; then
+  echo "Installing foreman..."
+  gem install foreman
+fi
+
+exec foreman start -f Procfile.dev "$@"
```
Comment (RasimKhusaenov) [inline bin/dev:8]: Do you need this file?

File: db/migrate/20230314211757_create_posts.rb [hunk 7]
```diff
@@ -1,6 +1,9 @@
+# frozen_string_literal: true
+
 class CreatePosts < ActiveRecord::Migration[7.0]
   def change
     create_table :posts do |t|
+      t.string :text
```
Comment (RasimKhusaenov) [inline db/migrate/20230314211757_create_posts.rb:6]: Do you want to add `null: false` constraint here?

File: db/migrate/20230314211744_create_users.rb [hunk 8]
```diff
@@ -1,6 +1,10 @@
+# frozen_string_literal: true
+
 class CreateUsers < ActiveRecord::Migration[7.0]
   def change
     create_table :users do |t|
+      t.string :name
+      t.string :surname
```
Comment (RasimKhusaenov) [inline db/migrate/20230314211744_create_users.rb:7]: Do you want to add `null: false` constraint here?

File: test/application_system_test_case.rb [hunk 9]
```diff
@@ -1,4 +1,6 @@
-require "test_helper"
```
Comment (RasimKhusaenov) [inline test/application_system_test_case.rb:1]: Do you need `test/` folder?

File: Procfile.dev (hunk 2 above)
Comment (RafaelKhammadeev) [reply Procfile.dev:2]: Now I don't think it's needed.

File: Gemfile (hunk 1 above)
Comment (RafaelKhammadeev) [reply Gemfile:64]: Okay, I chosen pry

File: app/views/application/_navigation.html.erb (hunk 5 above)
Comment (RafaelKhammadeev) [reply app/views/application/_navigation.html.erb:10]: In this PR, it's don't needed

File: bin/dev (hunk 6 above)
Comment (RafaelKhammadeev) [reply bin/dev:8]: No, I needed, maybe this file generate, when i try install bootstrap

File: Gemfile (hunk 3 above)
Comment (RafaelKhammadeev) [reply Gemfile:1]: No

**Review by RasimKhusaenov — APPROVED** [review]

**Pull request merged**